contrib/oc-environment.sh

```diff
@@ -31,6 +31,9 @@ export BRIDGE_K8S_MODE_OFF_CLUSTER_THANOS
 BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER=$(oc -n openshift-config-managed get configmap monitoring-shared-config -o jsonpath='{.data.alertmanagerPublicURL}')
 export BRIDGE_K8S_MODE_OFF_CLUSTER_ALERTMANAGER
 
+BRIDGE_K8S_MODE_OFF_GITOPS=$(oc -n openshift-pipelines-app-delivery get route cluster -o jsonpath='{.spec.host}')
```

Comment:
This will show an error every time we run `oc-environment.sh` and GitOps isn't set up. Is there a way we can avoid that?

Reply:
Can we use `2> /dev/null` to redirect the error when the GitOps isn't set up?